tools-local/tasks/mobile.tasks/AppleAppBuilder/Utils.cs

```diff
@@ -92,4 +92,10 @@ public static void LogError(string? msg)
             Logger?.LogError(msg);
         }
     }
+
+    internal class TargetOS
+    {
+        public static string iOS { get { return "iOS"; } }
+        public static string tvOS { get { return "tvOS"; } }
```

Comment:
looks like a constant so should be `const` 😄
```suggestion
        public const string iOS = "iOS";
        public const string tvOS = "tvOS";
```